client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/steps/use-steps.tsx

```diff
@@ -0,0 +1,146 @@
+import { ExternalLink } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import type { Task, Expandable } from '@automattic/launchpad';
+
+const removeDuplicatedSlashes = ( url: string ) => url.replace( /(https?:\/\/)|(\/)+/g, '$1$2' );
+
+const getPluginInstallationPage = ( fromUrl: string ) => {
+	if ( fromUrl !== '' ) {
+		return removeDuplicatedSlashes(
+			`${ fromUrl }/wp-admin/plugin-install.php?s=%2522migrate%2520guru%2522&tab=search&type=term`
+		);
+	}
+
+	return 'https://wordpress.org/plugins/migrate-guru/';
+};
+
+const getMigrateGuruPageURL = ( siteURL: string ) =>
+	removeDuplicatedSlashes( `${ siteURL }/wp-admin/admin.php?page=migrateguru` );
+
+const recordInstructionsLinkClick = ( linkname: string ) => {
+	recordTracksEvent( 'calypso_site_migration_instructions_link_click', {
+		linkname,
+	} );
+};
+
+interface Options {
+	fromUrl: string;
+}
+
+interface Step {
+	task: Task;
+	expandable?: Expandable;
+}
+
+type Steps = Step[];
+
+export const useSteps = ( { fromUrl }: Options ): Steps => {
+	const translate = useTranslate();
+
+	const steps = [
+		{
+			title: translate( 'Install the Migrate Guru plugin' ),
```

Comment:
The copy review has these steps numbered which I kinda like.

Reply:
I liked the numbers too but it presents some align issues with the child content for the step. We thought it would be better to leave the numbers out vs. updating the component for this one off case.

Reply:
It's something I discussed with @fditrapani. If we had the numbers, he would prefer having a separate element for the numbers (_"1."_), having the perfect alignment between the text and the expandable content. I wanted to avoid a bigger change in the original `ChecklistItem` component to avoid unexpected issues, so I suggested only a padding in the expandable, but he thought it would be better to remove the number in this case.

I think maybe my explanation is confusing to understand, but if you check the final options in the Figma file, it probably gets clearer. :)